app/src/main/kotlin/com/simplemobiletools/camera/implementations/QRScanner.java

```diff
@@ -56,7 +56,7 @@
     private AlertDialog alert = null;
 
     /** Scheduling flags */
-    public static boolean qr_requested = true;
+    public static boolean qr_requested = false;
```

Comment:
any reason for changing this? Not so familiar with the QR scanner code so just wondering.

Reply:
yes in PhotoProcessor.kt, we have this code when attempting to save a picture
![Screen Shot 2019-04-06 at 10 44 01 AM](https://user-images.githubusercontent.com/21287072/55671006-032b1700-5859-11e9-8e85-20bdd0fd6d0a.png)
when the _first picture_ gets taken it gets in that conditional even when we didn't request the scanner.
I'm pretty sure I checked everything and it was safe to set it to false initially, correct me if I'm wrong @aa-software2112

Reply:
that's odd, it should be false to begin with; you're right